apps/api_accounts/test/api_accounts_test.exs

```diff
@@ -123,6 +176,21 @@ defmodule ApiAccountsTest do
       assert ApiAccounts.list_keys_for_user(user) == []
     end
 
+    test "delete_tenable_user/1 deletes the user wasscan tenable" do
+      valid_users = list_of_valid_users()
+      valid_and_invalid_users = add_invalid_users()
+      assert ApiAccounts.list_users() == valid_and_invalid_users
+
+      for user <- valid_and_invalid_users do
+        for _ <- 1..5 do
+          {:ok, _} = ApiAccounts.create_key(user)
```

Comment:
Since you're creating keys for the users (which is a good idea), can you add an assertion that the keys are deleted as well?

Reply:
Added :+1: